src/ConsoleRunner/nunit-console/ConsoleOptions.cs

```diff
@@ -55,6 +56,9 @@ public class ConsoleOptions : CommandLineOptions
 		[Option(Description = "List of categories to exclude")]
 		public string exclude;
 
+        [Option(Short = "compat", Description = "Produce NUnit3 Compatibility Report")]
+        public bool compatibility;
+
```

Comment:
[File mixes tabs and spaces. Having tabs is the reason the diff looks so weird.]